tests/suites/test_suite_cmac.data

```diff
@@ -17,7 +17,7 @@ CMAC init #3 AES-256: OK
 depends_on:MBEDTLS_AES_C
 mbedtls_cmac_setkey:MBEDTLS_CIPHER_AES_256_ECB:256:0
 
-CMAC init #4 3DES : OK
+CMAC init #4 3DES: OK
```

Comment:
The commit message says `Remove unnecessary AES & CBC dependency` but the commit only changes whitespace. perhaps just ditch that commit the next time you rewrite history?